### DISCUSSION TITLE
Update for "Current Status - January 2023"

### DISCUSSION THREAD

**Discussion by richb-hanover** [PRQL/prql#1341] [General]
Proposed text for an update to the README.md... Mentioning the date included in the 0.4 Milestone would be optional.

----------
## Current Status - January 2023 _(Sigh... I have to learn to type a whole new year... :-)_

PRQL is being actively developed by a growing community. It's ready to use by the intrepid, either as part of one of our supported extensions, or within your own tools, using one of our supported language bindings.

PRQL continues to evolve toward the [0.4 Milestone.](https://github.com/PRQL/prql/milestone/4) The [CHANGELOG.md](https://github.com/PRQL/prql/blob/main/CHANGELOG.md) gives more information.

PRQL still has some minor bugs and some missing features, and probably is only ready to rolled out to non-technical teams for fairly simple queries. We're exploring where to focus further development; we welcome use-cases. Here's our current [Roadmap](https://prql-lang.org/roadmap/) and our longer-term [set of Milestones.](https://github.com/PRQL/prql/milestones)

----------
As I paw through the list of closed PRs, I realize that a huge amount of work has already occurred. The CHANGELOG.md should mention the following items (along with anything else that's interesting):

- feat: Playground queries sample database (#1305)
- feat: comment that displays the compiler version used to generate the SQL (#1322)
- feat: switch (#1278)
- feat: `in` operator (#1330)
- feat: Allow pipelines in list items (#1318)
- feat: Parsing for negative select (#1317)
- feat: additional builtin functions (#1325)
- fix: allow interpolations in table s-strings (#1337)

... and link to documentation pages that describe how to use the feature.

Thanks, and congratulations!

**Comment by max-sixty** [comment]
Great list, thank you @richb-hanover !

To the extent you want to PR both the update and the Changelog, that would be a nice contribution...
